### PR TITLE
Feature backend(global) 41 implementing search bar functionality

### PR DESCRIPTION
**Pull request by ValantineSuh**
Implementing Search Bar Functionality

### CODE REVIEW

File: instance/OIS.db
Comment (stephane-segning) [inline instance/OIS.db:1]: This file is a SQLite file. You never want this in your VCS. Good work here

**Review by Koufan-De-King — APPROVED** [review]
Good work, keep it up.

File: instance/OIS.db
Comment (ValantineSuh) [reply instance/OIS.db:1]: Thank you Sir @stephane-segning

**Review by Awambeng — APPROVED** [review]
good

**Review by stephane-segning — APPROVED** [review]
Good work overall. jQuery is a simple and effective library for working on the Internet with APIs and manipulating the DOM. Play around with it more, but don't forget that it is now deprecated in favour of newer frameworks, such as SPA.

**Pull request merged**